intezer_sdk/alerts.py

```diff
@@ -75,6 +92,13 @@ def __init__(self, alert_id: str, api: IntezerApiClient = None):
         self.status: Optional[AlertStatusCode] = None
         self.scans: List[Union[UrlAnalysis, FileAnalysis, EndpointAnalysis]] = []
 
+    @classmethod
+    def _parse_alert_id_from_alert_stream(cls, alert_stream: BinaryIO) -> str:
+        try:
+            return hashlib.sha256(alert_stream.read()).hexdigest()
```

Comment:
let's validate somewhere we didn't got an empty stream

Reply:
done